app/src/main/res/values/array.xml

```diff
@@ -54,6 +54,7 @@
         <item>@string/player_settings_play_in_app</item>
         <item>@string/player_settings_play_in_vlc</item>
         <item>@string/player_settings_play_in_mpv</item>
+        <item>@string/player_settings_play_in_mpvytdl</item>
```

Comment:
You forgot to add 6 in player_pref_values, right now you get "No app found" because it looks at the fourth position in player_pref_values being 4 = "Web Video Cast"